### ISSUE TITLE
Messages - New 1:1 chat screen issues

### ISSUE THREAD

**Issue by Francesca-G** [status-im/status-react#19266]
# Design Feedback 2.27

## [Messages] New 1:1 chat screen issues
Figma frame of the Review [here](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj/Release-Reviews?type=design&node-id=7%3A111976&mode=design&t=hS39AbDcBnA1KjAm-1)

- **Missing shadow** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#729029531)] of the content card
- **New contact avatar** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#725938513)] isn't visible, logged here https://github.com/status-im/status-mobile/issues/18686
- **Contact bio** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#727179051)] isn't visible (the bio was present and visible in the profile settings)
- **_No pinned messages_ action** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#725937293)] should adapt to screen width, keeping the _Mute chat_ action a square
- **Spacing** [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#725937654)] between Actions and Date divider doesn't match design

<img width="500" alt="Screenshot 2024-03-15 alle 10 53 08" src="https://github.com/status-im/status-mobile/assets/122543089/7a575968-88ac-4ca0-9740-96650b85ca84">

### Reproduction

1. Add a new contact
2. Open the chat

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS
- Devices in the screenshots: iPhone 15 Pro Max

[comment]: # (Please, add logs/notes if necessary)

**Comment by flexsurfer** [comment]
- [x] Missing shadow [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#729029531)] of the content card
- [x] New contact avatar [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#725938513)] isn't visible, logged here Profile photo of a new contact is not dispayed after accepting contact request  #18686
- [ ] Contact bio [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#727179051)] isn't visible (the bio was present and visible in the profile settings) (blocked by https://github.com/status-im/status-mobile/pull/19237)
- [x] No pinned messages action [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#725937293)] should adapt to screen width, keeping the Mute chat action a square (will be fixed in another PR https://github.com/status-im/status-mobile/pull/19245)
- [x] Spacing [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#725937654)] between Actions and Date divider doesn't match design (will be fixed here https://github.com/status-im/status-mobile/pull/19015)

**Comment by flexsurfer** [comment]
Contact bio [[comment](https://www.figma.com/file/oznddnBnDbLlLQIaACYNuj?node-id=94:35482&mode=design#727179051)] isn't visible (the bio was present and visible in the profile settings)

**Comment by mohsen-ghafouri** [comment]
Bio is already available in 1-1 chat, @Francesca-G could you please confirm and we can close this issue

<img src="https://github.com/status-im/status-mobile/assets/71308738/56130e45-41ce-495a-8e0a-92fbf4818e1d" width="390px"/>

**Comment by Francesca-G** [comment]
> Bio is already available in 1-1 chat, @Francesca-G could you please confirm and we can close this issue
> 
Yes 👍 
looks fixed!
